equipment.py

```diff
@@ -271,9 +262,11 @@
             },
             "Domestic_Hot_Water_Valve": {
                 "tags": [TAG.Domestic, TAG.Water, TAG.Hot, TAG.Valve, TAG.Heat, TAG.Equipment],
+                "parents": [BRICK.Hot_Water_System, BRICK.Water_Valve],
```

Comment:
I am not sure if ``Hot_Water_System`` is a superclass of everything related to ``Hot_Water``. I know we haven't precisely defined it but I always thought it is a collection of entities for controlling hot water like ``AHU`` for air handling. For example, ``Supply_Fan`` is not a subclass of ``AHU``. I "feel" a ``Valve`` is a part of ``Water_System`` likewise, but  I can't strongly argue it as we don't have the right definition. If you feel that the current hierarchy is correct, we can discuss this later if somebody complains.

Reply:
Fair point! `Supply_Fan` is not a subclass of `AHU`, but both are subclasses of `HVAC` equipment; I feel this is similar. `Hot_Water_Valve` is similarly component of the `Hot_Water_System`. Our class structure is consistent with this pattern for now; if we want to change it in the future, that could be its own pull request